packages/gatsby/src/utils/page-ssr-module/entry.ts

```diff
@@ -28,6 +28,7 @@ export interface ISSRData {
   page: IGatsbyPage
   templateDetails: ITemplateDetails
   potentialPagePath: string
+  serverDataHeaders?: Record<string, string | number | Array<string>>
```

Comment:
This should always be a string. Headers can only hold string values, if they hold anything else it get converted
```suggestion
  serverDataHeaders?: Record<string, string>
```

Reply:
I followed https://microsoft.github.io/PowerBI-JavaScript/classes/_node_modules__types_node_http_d_._http_.outgoingmessage.html#setheader which is signature for function I later set headers on `res` in `gatsby serve`. I'm fine with the change - just checking if you are sure if it's currect one?